analytics-next/index.js

```diff
@@ -50,3 +50,19 @@ function setUserProperties() {
   setUserProperties(analytics, { favorite_food: 'apples' });
   // [END analytics_set_user_properties]
 }
+
+function recordScreenView() {
+  const screenName = '<SCREEN_NAME>';
+
+  // [START analytics_record_screen_view]
+  const { getAnalytics, logEvent } = require("firebase/analytics");
+
+  const analytics = getAnalytics();
+  logEvent(analytics, 'screen_view', {
+    app_name: 'My App',
```

Comment:
@hsubox76 all 4 of these params seem to be required by the `logEvent()` type definition. Is that correct?